service/controller/v1/key/key.go

```diff
@@ -0,0 +1,45 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	VERSION_BUNDLE = "giantswarm.io/version-bundle"
```

Comment:
```suggestion
 	versionBundleLabel = "giantswarm.io/version-bundle"
```

Reply:
Actually I think this should be `versionBundleAnnotation`. 😆 Missed it as this is one of our first uses of annotations!